xbmc/platform/darwin/osx/network/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(SOURCES NetworkOsx.cpp)
-set(HEADERS NetworkOsx.h)
+set(SOURCES NetworkmacOS.mm)
```

Comment:
Should probably be NetworkMacOS.mm. "macOS" isn't always a lower case M. When the proper noun starts a sentence then it's capitalized as "MacOS". I would think CamelCase would also capitalize MacOS.